docs/build.gradle

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.elasticsearch.gradle.SnippetsTask
+import org.elasticsearch.gradle.SnippetsTask.Snippet
+import org.elasticsearch.gradle.RestTestsFromSnippetsTask
+
+apply plugin: 'elasticsearch.rest-test'
+
+task(listSnippets, type: SnippetsTask) {
```

Comment:
In all our other gradle files we use the form `task NAME(ARGS) {` for defining tasks